pkg/resources/server.go

```diff
@@ -5,34 +5,97 @@ import (
 	"github.com/Mellanox/k8s-rdma-shared-dev-plugin/pkg/types"
```

Comment:
please organize the imports see [1] 
you can use [2] to organize the imports.
Usually it divide:
1. standard imports
2. third party imports
3. project imports

[1] - https://github.com/golang/go/wiki/CodeReviewComments#imports 
[2] - https://godoc.org/golang.org/x/tools/cmd/goimports

Reply:
fixed